state_solver/suspension.py

```diff
@@ -60,11 +70,11 @@ def get_loads(self, vehicle_velocity:np.array, yaw_rate:float, steered_angle:flo
             self.logger.log(tire_name + "_RLCA_force", tube_forces[3])
             self.logger.log(tire_name + "_pullrod_force", tube_forces[4])
             self.logger.log(tire_name + "_toe_link_force", tube_forces[5])
-            
-        return forces, moments
+
+        return veh_forces, veh_moments, wheel_speeds, wheel_torques
```

Comment:
Just clarifying, but the suspension now handles solving for the wheel speeds as a function of all the MMM states and this eventually feeds into the diff?

Reply:
Not a fan of wheel_torques needing to be solved through suspension, but I can understand why you did it here. Is there a way the suspension can somehow modify the state of the tire class so that the diff can later access? Probably warrants a longer convo with the data structuring class we had proposed

Reply:
@kierancoz double checking you saw this one